css/styles.css

```diff
@@ -50,13 +50,59 @@ body {
   font-weight: 700;
 }
 
-.navigation-bar {
+.hamburger {
+  position: relative;
+  z-index: 1000;
+  top: 21%;
+  right: 1rem;
+  padding: 4px;
+  cursor: grabbing;
+  margin-bottom: 2rem;
+}
+
+.nav-menu {
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.6s ease-in-out;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ffe6;
+  backdrop-filter: blur(2px);
+  list-style: none;
+  padding-top: 6rem;
+}
+
+.nav-menu li a {
+  color: var(--color-body);
+  text-decoration: none;
+}
+
+i.cross-icon.fa-solid.fa-times {
   display: none;
+  position: fixed;
+  color: var(--white);
 }
 
-.fa-bars {
+.list-item {
+  font-size: 3rem;
+  padding: 3rem;
+}
```

Comment:
Kindly use the `font-weight` as in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546) so that your list items are bolder:

Yours | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/38088818/167595293-538d54e2-1414-473f-b2f9-227e37b63bf8.png)  | ![image](https://user-images.githubusercontent.com/38088818/167594605-805488f5-f927-41ae-a2da-0bb3b938882f.png)

Because you used padding to create spaces in between the menu items. The borders overlap each other making it hard to click on the correct item. Sometimes when I click `portfolio` it takes me to about, and when I click `about` it takes me to contact.
![image](https://user-images.githubusercontent.com/38088818/167596176-545c3400-89fe-4e4f-af7d-636d6860f8a0.png)

- If I add a red border on the items for illustration, this is how they overlap:
![image](https://user-images.githubusercontent.com/38088818/167596660-4c216255-cdbb-4953-9716-063dba90bebf.png)

Kindly use margins to space the menu items' top and bottom spaces instead of padding so the borders don't overlap like this:
![image](https://user-images.githubusercontent.com/38088818/167597981-7140a1fb-f1cc-43e4-91eb-0b7b2e9cf27f.png)

You can set the top and right padding to zero while keeping the right and left as `3rem`:
![image](https://user-images.githubusercontent.com/38088818/167598512-fae2594e-44ea-42ac-8631-8508a0bb8c29.png)

Then apply only top and bottom margins and keep the left and right as zero since you already have the padding for left and right. However, make sure you apply the margins on the `li` tag, not the `a` tag you classed as `list-item` otherwise the margin will not apply since the `a` tag is a child of the `li` tag and you need the margins to be between the `li` tags.

For example:
![image](https://user-images.githubusercontent.com/38088818/167599725-3badc956-0b9a-4597-8e0d-156425e6c15c.png)

Reply:
Thank you.